utils/network.go

```diff
@@ -22,7 +22,7 @@ type MockOptions struct {
 	DesiredEndpoint string
 }
 
-func TcpCheck(hp *HostPort, mock *MockOptions) bool {
+func TCPCheck(hp *HostPort, mock *MockOptions) bool {
```

Comment:
This is a breaking change. Not sure we should change that. @kumarabd ?
It can be exclude from the linter by using a comment like `//nolint:<check>`.